launch/tier4_localization_launch/config/ndt_scan_matcher.param.yaml

```diff
@@ -65,3 +65,9 @@
 
     # regularization scale factor
     regularization_scale_factor: 0.01
+
+    # A flag for using scan matching score based on de-grounded LiDAR scan
+    estimate_scores_for_degrounded_scan: false
+
+    # If lidar_point.z - base_link.z <= this threshold , will be removed
+    ground_removal_z_threshold: 0.8
```

Comment:
If the threshold is 0.8m, it seems to include objects other than the ground.
So I think it would be better to change the word to mean something of low height instead of saying ground.

Reply:
I'm not a native English speaker, so I can't think of a good word for it.

Reply:
I just want to remove the ground, I found that if this threshold is set too small can not remove the ground.

Reply:
I think what he meant was not about the value but about the variable name. Still cannot come up with a right fit vocabulary, but how about something like this?
- `margin_from_base_link`
- `z_margin_for_ground_removal`